templater/jobs/presubmit/eks-anywhere-build-tooling/cfssl-presubmits.yaml

```diff
@@ -0,0 +1,13 @@
+jobName: cfssl-tooling-presubmit
```

Comment:
This job can be removed, I actually thought @pokearu would have done that already...